Orange/widgets/unsupervised/owcorrespondence.py

```diff
@@ -61,6 +61,7 @@ class Inputs:
 
     class Error(widget.OWWidget.Error):
         empty_data = widget.Msg("Empty data set")
+        no_disc_vars = widget.Msg("Correspondence Analysis needs categorical variables")
```

Comment:
Add a period at the end (we follow the rule that if the message is a complete sentence, we add a period).

Alternatively, shorten this to `"No categorical variables"` (without a period).